src/cluster_slot_stats.c

```diff
@@ -113,6 +125,22 @@ static void addReplyOrderBy(client *c, int order_by, long limit, int desc) {
     addReplySortedSlotStats(c, slot_stats, limit);
 }
 
+/* Resets applicable slot statistics. */
+void clusterSlotStatReset(int slot) {
+    /* key-count is exempt, as it is queried separately through `countKeysInSlot()`. */
+    cluster_slot_stats[slot].cpu = 0;
```

Comment:
Would this be better to avoid updating the method repetitively?

```suggestion
    memset(&cluster_slot_stats[slot], 0, sizeof(slotStat));
```

Reply:
Agreed, I will switch to `memset()` instead.

My original thought process was to explicitly reset per-metric, since the upcoming metric `memory-bytes` should not be reset upon `CONFIG RESETSTAT`. But this is unnecessary forward thinking for now, and can be revisited once `memory-bytes` PR is opened.